doc/tutorials/using_hybrid_mode_on_nuc.rst

```diff
@@ -143,6 +162,9 @@ Perform the following to update Ubuntu GRUB so it can boot the hypervisor and lo
 
 Hybrid Scenario Startup Check
 *****************************
+#. Refer to the **Test the Communication Port** section in the :ref:`vuart_config` tutorial. 
```

Comment:
There is no such command. The tutorial provides instructions to test that the serial port connection but we don't need to repeat these here.

The https://projectacrn.github.io/latest/tutorials/using_serial_port.html would be a better tutorial to reference here actually:
```suggestion
#. Refer to the **Test the Communication Port** section in the :ref:`connect_serial_port` tutorial. 
```